bika/lims/browser/clientfolder.py

```diff
@@ -39,95 +37,94 @@ def __init__(self, context, request):
+        self.landing_page = get_registry_value(
+            self._LANDING_PAGE_REGISTRY_KEY, self._DEFAULT_LANDING_PAGE)
 
-        self.filter_indexes = ['title', 'SearchableText']
+        self.contentFilter = {
+            "portal_type": "Client",
+            "sort_on": "sortable_title",
+            "sort_order": "ascending"
+        }
 
         self.show_sort_column = False
         self.show_select_row = False
         self.show_select_all_checkbox = False
         self.show_select_column = False
         self.pagesize = 25
-        self.icon = '{}/{}'.\
-            format(self.portal_url,
-            "++resource++bika.lims.images/client_big.png")
-        request.set('disable_border', 1)
-
-        self.columns = {
-            'title': {'title': _('Name'), 'index': 'sortable_title'},
-            'EmailAddress': {'title': _('Email Address')},
-            'getCountry': {'title': _('Country')},
-            'getProvince': {'title': _('Province')},
-            'getDistrict': {'title': _('District')},
-            'Phone': {'title': _('Phone')},
-            'Fax': {'title': _('Fax')},
-            'ClientID': {'title': _('Client ID')},
-            'BulkDiscount': {'title': _('Bulk Discount')},
-            'MemberDiscountApplies': {'title': _('Member Discount')},
-        }
-
-        self.review_states = [  # leave these titles and ids alone
-            {'id': 'default',
-             'contentFilter': {'inactive_state': 'active'},
-             'title': _('Active'),
-             'transitions': [{'id': 'deactivate'}, ],
-             'columns': ['title',
-                         'ClientID',
-                         'getCountry',
-                         'getProvince',
-                         'getDistrict',
-                         'EmailAddress',
-                         'Phone',
-                         'Fax', ]
-             },
-            {'id': 'inactive',
-             'title': _('Dormant'),
-             'contentFilter': {'inactive_state': 'inactive'},
-             'transitions': [{'id': 'activate'}, ],
-             'columns': ['title',
-                         'ClientID',
-                         'getCountry',
-                         'getProvince',
-                         'getDistrict',
-                         'EmailAddress',
-                         'Phone',
-                         'Fax', ]
-             },
-            {'id': 'all',
-             'title': _('All'),
-             'contentFilter': {},
-             'transitions': [],
-             'columns': ['title',
-                         'ClientID',
-                         'getCountry',
-                         'getProvince',
-                         'getDistrict',
-                         'EmailAddress',
-                         'Phone',
-                         'Fax', ]
-             },
+        self.icon = "{}/{}".format(
+            self.portal_url, "++resource++bika.lims.images/client_big.png")
```

Comment:
You could use this function https://github.com/senaite/senaite.core/blob/master/bika/lims/utils/__init__.py#L797-L808 
```python
self.icon = get_image("client_big.png")
```